tests/testthat/test-time_binning.R

```diff
@@ -0,0 +1,49 @@
+test_that("time_binning() works", {
+
+  #error handling
+  expect_error(time_binning(occdf = c(50, 20, 10)))
+
+  expect_error(time_binning(bins = c(50, 20, 10)))
+
+  expect_error(time_binning(occdf = data.frame(),
+                            bins = c(50, 20, 10)))
+
+  expect_error(time_binning(occdf = data.frame(),
+                            bins = data.frame(),
+                            method = "assign"))
+
+  expect_error(time_binning(occdf = data.frame(),
+                            bins = data.frame(),
+                            method = "mid",
+                            reps = TRUE))
+
+  expect_error(time_binning(occdf = data.frame(),
+                            bins = data.frame(),
+                            method = "mid",
+                            scale = "2012"))
+
+  expect_error(time_binning(occdf = data.frame(),
+                            bins = data.frame(),
+                            method = "mid",
+                            scale = "GTS2020",
+                            return_error = "TRUE"))
+
+  #expect equal
+  occdf <- read.csv("https://paleobiodb.org/data1.2/colls/list.csv?base_name=Scleractinia")
```

Comment:
If you're going to depend on the pbdb, you should put a check in to skip the rest of these if internet or the site is down (see `skip_if_offline()`)

Reply:
Thanks Will, I actually plan to remove all instances of this from tests and examples. It is just a placeholder until we get the example occurrence data from Emma and Bethany for the package.